std/datetime/package.d

```diff
@@ -1,98 +1,76 @@
 // Written in the D programming language
 
 /++
-    Module containing Date/Time functionality.
+    $(SCRIPT inhibitQuickIndex = 1;)
 
-    This module provides:
-    $(UL
-        $(LI Types to represent points in time:
-             $(REF SysTime,std,_datetime,systime),
-             $(REF Date,std,_datetime,date),
-             $(REF TimeOfDay,std,_datetime,date),
-             $(REF DateTime,std,_datetime,date).)
-        $(LI Types to represent intervals of time.)
-        $(LI Types to represent ranges over intervals of time.)
-        $(LI Types to represent time zones (used by
-             $(REF SysTime,std,_datetime,systime)).)
-        $(LI A platform-independent, high precision stopwatch type:
-             $(LREF StopWatch))
-        $(LI Benchmarking functions.)
-        $(LI Various helper functions.)
+    Phobos provides the following functionality for time:
+
+    $(DIVC quickindex,
+    $(BOOKTABLE ,
+    $(TR $(TH Functionality) $(TH Symbols)
+    )
+    $(TR
+        $(TD Points in Time)
+        $(TD
+            $(REF_ALTTEXT Date, Date, std, _datetime, date)$(NBSP)
```

Comment:
Preventing auto-highlighting is no longer needed :)
see e.g. https://github.com/dlang/dlang.org/pull/2307